src/app/ipv/middleware.js

```diff
@@ -210,6 +211,40 @@ function checkJourneyAction(req) {
   }
 }
 
+// getCoiUpdateDetailsJourney determines the next journey based on the detailsToUpdate
+// field of the update-details page
+function getCoiUpdateDetailsJourney(detailsToUpdate) {
+  if (detailsToUpdate) {
+    // convert to array if its a string
+    if (typeof detailsToUpdate === "string") {
+      detailsToUpdate = [detailsToUpdate];
+    }
+    // cancel event
```

Comment:
Sorry to nit pick, but I think the if cases for some of these comments might be clear enough already (and I think we frequently fail to keep them up to date anyway) 🙈